ivy/functional/ivy/random.py

```diff
@@ -344,41 +357,220 @@ def random_normal(
      ret
         Drawn samples from the parameterized normal distribution.
 
-    Funtional Examples
-    ------------------
+    Functional Examples
+    -------------------
 
-    >>> y = ivy.random_normal(0.0, 2.0)
-    >>> print(y)
-    ivy.array(0.6444774682897879)
+    >>> ivy.random_normal()
+    ivy.array(-0.22346112)
 
-    >>> y = ivy.random_normal(shape=3)
-    >>> print(y)
-    ivy.array([ 0.811, -0.508, -0.564])
+    >>> ivy.random_normal(shape=3)
+    ivy.array([-0.73  ,  0.0922, -0.515 ])
 
-    >>> y = ivy.random_normal(0.0,2.0,device='cpu')
-    >>> print(y)
-    ivy.array(-0.7268672)
+    >>> ivy.random_normal(shape=(2,3))
+    ivy.array([[-0.361 ,  0.596 , -0.247 ],
+               [-1.39  ,  0.0426, -0.627 ]])
 
-    >>> y = ivy.random_normal(0.7, 1.0, device="cpu", shape=(2, 2))
-    >>> print(y)
-    ivy.array([[1.17 , 0.968],
-               [0.175, 0.064]])
+    >>> ivy.random_normal(3.0, 6.0)
+    ivy.array(4.9213753)
+
+    >>> ivy.random_normal(1.0, 2.0, (2,1))
+    ivy.array([[2.19],
+               [2.78]])
+
+    >>> z = ivy.zeros(())
+    >>> ivy.random_normal(1.0, 2.0, out=z)
+    ivy.array(0.12818667)
+
+    >>> ivy.random_normal(1.0, 2.0, (2,2), device='cpu')
+    ivy.array([[ 2.91 ,  1.3  ],
+               [ 3.37 , -0.799]])
+
+    >>> ivy.random_normal(1.0, 2.0, (2,2), device='cpu', dtype='int32')
+    ivy.array([[ 0, -1],
+               [ 0,  3]])
+
+    >>> z = ivy.zeros((1,2))
+    >>> ivy.random_normal(1.0, 2.0, (1,2), device='cpu', dtype='float64', out=z)
+    ivy.array([[-2.01, -1.95]])
+
+    >>> x = ivy.array([4.8, 5.6])
+    >>> y = ivy.array([9.8, 7.4])
+    >>> ivy.random_normal(x, y)
+    ivy.array([ 4.43 , -0.469])
+
+    >>> z = ivy.zeros((2,))
+    >>> ivy.random_normal(x, y, out=z)
+    ivy.array([0.287, 8.55 ])
+
+    >>> ivy.random_normal(x, y, device='cpu')
+    ivy.array([18.9, 15.2])
+
+    >>> ivy.random_normal(x, y, device='cpu', dtype='float64')
+    ivy.array([-4.1   , -0.0366])
+
+    >>> z = ivy.zeros((2,))
+    >>> ivy.random_normal(x, y, device='cpu', dtype='float64', out=z)
+    ivy.array([12.4, 11. ])
 
     Instance Method Examples
     ------------------------
```

Comment:
don't need these examples in the functional API docstring, check [Docstring Examples](https://github.com/unifyai/ivy/blob/517e457514defdb36af1eef135c5fc8e20470c5f/docs/partial_source/deep_dive/13_docstring_examples.rst) section.